report.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sbn
+import json
+
+
+class ReportAssistant():
+    def __init__(self):
+        self.usage = pd.read_csv("./state/metrics/usage.csv")
+        # self.messages = pd.read_csv("./state/metrics/messages.csv")
+        self.feedback = pd.read_csv("./state/metrics/feedback.csv")
+        self.current_df = None
```

Comment:
This smells like trouble. Let's chat about the design here (in office hours or in class).